### DISCUSSION TITLE
I think the trial is too short to be useful

### DISCUSSION THREAD

**Discussion by Patabugen** [spatie/ray#381] [Q&A] [answered]
The trial gives us 10 messages (one of which it tells us to use as part of the setup instructions).

I installed Ray (with Composer), and before I'd even typed a `ray()` apparently I've used up 11. The licence screen hides any output so I can neither see what they were to understand how I've already spent my trial, or use whatever messages they were to try out using Ray. The trial has essentially finished before it's begin.

The trial limitation of 10 seems incredibly limited and I wanted to suggest increasing it (perhaps to 100).

The [Introduction lists 5 examples](https://spatie.be/docs/ray/v1/introduction) that I'd like to try out, and the [homepage lists 8](https://myray.app/).

This means that in the trial I can either test out a few of the examples you give, or I can blindly test it out in my app. But there's not enough allowance to both try it out in our apps and also see what features it offers, let alone try anything past the first page of the documentation.

I just thought I'd feedback, Ray looks really cool - but the trial feels disconcertingly pointless.

**Comment by freekmurze** [comment, marked as answer]
If you quit and restart the app, the limit will reset. So you can try out some more stuff if you want.

**Reply by Patabugen** [reply]
Ah, thanks! That's really handy!

**Reply by Patabugen** [reply]
Ah, I've come across another (potential) problem!

The issue I'm hitting now is that the app locks after 10 items have been received. So I need to be careful to send fewer than 10 if I want to be able to see how the app works (since all usage is blocked once the 10th is reached).

I notice it automatically picks up on my Laravel log calls which I think is what's hitting the limit - so I have to turns those off in order to trial the app.

**Comment by lcallif** [comment]
It is stated that the trial is for 14 days, but we have the following message: In trial mode you can send up to 10 log items before the app locks.

Why do the lock if we have a 14 day trial ?  I would like to put Ray thru its paces before I purchase it, again.  I am a tad shy to plunk down the money if I encounter anything like the memory issue with the previous version.

Sooooooo, I would like to suggest that the free 14 day trial be free of the 10 log item limit.  Ok, not suggest, request.  Please ?

**Reply by freekmurze** [reply]
I don't think we have a 14 day limit, just the log limit of 10 items.

In the upcoming v3, we'll up the limit to 20. https://next.myray.app

**Reply by DriesH** [reply]
v3.0.0-beta.19 has an increased log limit (max 20)
